android/demos/compose/src/main/kotlin/com/uber/rib/compose/root/main/MainInteractor.kt

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (C) 2021. Uber Technologies
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.uber.rib.compose.root.main
+
+import com.uber.autodispose.autoDispose
+import com.uber.rib.core.BasicInteractor
+import com.uber.rib.core.Bundle
+import com.uber.rib.core.ComposePresenter
+import io.reactivex.android.schedulers.AndroidSchedulers.mainThread
+import io.reactivex.schedulers.Schedulers
+
+class MainInteractor(
+  presenter: ComposePresenter,
+  private val authStream: AuthStream,
+  private val childContent: MainRouter.ChildContent
+) : BasicInteractor<ComposePresenter, MainRouter>(presenter) {
+
+  override fun didBecomeActive(savedInstanceState: Bundle?) {
+    super.didBecomeActive(savedInstanceState)
+    router.view.setContent { MainView(childContent = childContent) }
```

Comment:
Why are we duplicating compose view create here?
We already have done it in local `ComposePresenter`.

Can't we just use `router.view.setContent { presenter.composable() }`?